test/lib/adminUtilsTests.test.js

```diff
@@ -0,0 +1,333 @@
+      it('should return an empty string for a valid zip code', async () => {
+        const result = await validateField('updateZipcode', '94612');
+
+        expect(result).to.be.a('string');
+        expect(result).to.equal('');
+      });
+      it('should return an error for a number that is not 5 digits', async () => {
+        const result = await validateField('updateZipcode', '946123');
+
+        expect(result).to.be.a('string');
+        expect(result).to.equal('Must be 5 digits');
+      });
+    });
+  });
+
+  describe('#removeOwner', () => {
+    const owner = {
+      id: 5,
+      projectGroupId: 0
+    };
+    const loggedInOwner = { id: 365 };
+    const projectGroup = {
+      id: 0,
+      ownerIds: [1, 2, 3, 4, 5]
+    };
+
+    beforeEach(() => {
+      sandbox.stub(request, 'getProjectGroupById').returns(projectGroup);
+      sandbox.stub(request, 'updateProjectGroup');
+      sandbox.stub(userData, 'refreshUserData');
+      sandbox.stub(store, 'getState').returns({ userData: loggedInOwner });
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should succesfully remove an owner from a project group', async () => {
+      await removeOwner(owner);
+
+      expect(request.updateProjectGroup.getCall(0).args[0]).to.equal(0);
+      expect(
+        JSON.stringify(request.updateProjectGroup.getCall(0).args[1].ownerIds)
+      ).to.equal(JSON.stringify([1, 2, 3, 4]));
+      expect(userData.refreshUserData.getCall(0).args[0]).to.equal(365);
+    });
+  });
+
+  describe('#getOwnerRecordsForProjectGroup', () => {
+    const owners = [
+      { id: 0, onboardingStep: 0 },
+      { id: 1, onboardingStep: -1 },
+      { id: 2, onboardingStep: -1 }
+    ];
+    const projectGroup = {
+      ownerIds: [0, 1, 2]
+    };
+
+    beforeEach(() => {
+      sandbox.stub(request, 'getOwnersByIds').returns(owners);
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should only get owners with onboardingStep of -1', async () => {
+      const result = await getOwnerRecordsForProjectGroup(projectGroup);
+
+      expect(JSON.stringify(result)).to.equal(
+        JSON.stringify([
+          { id: 1, onboardingStep: -1 },
+          { id: 2, onboardingStep: -1 }
+        ])
+      );
+    });
+  });
+
+  describe('#inviteMember', () => {
+    // TODO OP-24: make stock test objects for each object to use in places like this
+    const pledgeInvite = { foo: 'bar' };
+
+    beforeEach(() => {
+      sandbox.stub(request, 'createPledgeInvite');
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should create a pledge invite object', async () => {
+      await inviteMember(pledgeInvite);
+
+      expect(request.createPledgeInvite.getCall(0).args[0]).to.equal(
+        pledgeInvite
+      );
+    });
+  });
+
+  describe('#triggerEmail', () => {
+    // TODO OP-24: create dummy object for this
+    const pledgeInvite = { foo: 'bar' };
```

Comment:
@jassol because `Invite` is a data type that the application has defined, best practice would be to have a file that actually defines a dummy `Invite` object with the correct structure and with all non-null fields populated with dummy data that could be what you find in the database. That way it's easier to test data conversion (like string to datetime conversion) or dependencies between fields without having to define a new object in every test.